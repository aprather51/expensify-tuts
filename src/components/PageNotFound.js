import React from 'react';
import { Link } from 'react-router-dom';

const PageNotFound = () => (
    <div>
        404! Page Not Found <Link to="/">Return Back</Link>
    </div>
);

export default PageNotFound;