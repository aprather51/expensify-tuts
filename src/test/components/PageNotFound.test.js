import React from 'react'
import { shallow } from 'enzyme';
import PageNotFound from '../../components/PageNotFound';

test('Render Page Not Found Correctly', () => {
    const wrapper = shallow(<PageNotFound/>);
    expect(wrapper).toMatchSnapshot();
});
