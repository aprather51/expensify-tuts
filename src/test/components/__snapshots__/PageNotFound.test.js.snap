// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Page Not Found Correctly 1`] = `
<div>
  404! Page Not Found 
  <Link
    replace={false}
    to="/"
  >
    Return Back
  </Link>
</div>
`;
